src/main.ts

```diff
@@ -47,13 +54,19 @@ const command: Command = {
 
 		options('c', {
 			alias: 'config',
-			describe: 'Specifies what configuration to test with: \'local\'(default), \'browserstack\', \'testingbot\', or \'saucelabs\'.',
+			describe: `Specifies what configuration to test with: 'local'(default), 'browserstack', 'testingbot', or 'saucelabs'.`,
```

Comment:
Do you think this would read better as 'Specifies *the* configuration to test with: ...' ?